common/app/views/fragments/footer.scala.html

```diff
@@ -40,9 +40,34 @@
     </div>
 
     <div class="l-footer__secondary gs-container" role="contentinfo">
-        @if(!amp) {
-            <ul class="colophon u-cf">
-            @defining(Edition(request)) { currentEdition =>
+        <ul class="colophon u-cf">
+        @defining(Edition(request)) { currentEdition =>
+            @if(amp) {
+                @if(currentEdition == Uk) {
+                    <li class="colophon__item"><a data-link-name="uk : footer : contact us" href="@LinkTo {/help/contact-us}">
+                        contact us</a></li>
+                    <li class="colophon__item"><a data-link-name="uk : footer : jobs" href="http://jobs.theguardian.com/?INTCMP=NGW_FOOTER_UK_GU_JOBS">
+                        jobs</a></li>
+                }
+                @if(currentEdition == Us) {
+                    <li class="colophon__item"><a data-link-name="us : footer : contact us" href="@LinkTo {/info/about-guardian-us/contact}">
+                        contact us</a></li>
+                    <li class="colophon__item"><a data-link-name="us : footer : jobs" href="http://jobs.theguardian.com/?INTCMP=NGW_FOOTER_US_GU_JOBS">
+                        jobs</a></li>
+                }
+                @if(currentEdition == Au) {
+                    <li class="colophon__item"><a data-link-name="au : footer : contact us" href="@LinkTo {/info/2013/may/26/contact-guardian-australia}">
+                        contact us</a></li>
+                    <li class="colophon__item"><a data-link-name="au : footer : vacancies" href="@LinkTo {/info/2014/aug/11/guardian-australia-vacancies}">
+                        vacancies</a></li>
+                }
+                <li class="colophon__item"><a data-link-name="terms" href="@LinkTo {/help/terms-of-service}">
+                    terms &amp; conditions</a></li>
+                @if(currentEdition != Au) {
+                    <li class="colophon__item"><a data-link-name="complaints" href="@LinkTo {/info/complaints-and-corrections}">
+                        complaints &amp; corrections</a></li>
+                }
```

Comment:
Aware of the duplication, but thought it was the cleaner way of distinguishing what will display on AMP.